distribution/resources/conf/default-micro-gw.conf.template

```diff
@@ -52,6 +52,8 @@
   serverUrl = "https://localhost:9443"
   # The token endpoint context of the Key Manager server
   tokenContext = "oauth2"
+  # When Microgateway is used with older APIM versions for subscription validation by using KeyValidation service.
+  isLegacyKM = false
```

Comment:
Since we go with `enable` prefix for other configs, shall we rename this config to something like `enableLegacyMode`?

Reply:
Will fix this in the new branch